browser/about_flags.cc

```diff
@@ -426,15 +426,6 @@ const flags_ui::FeatureEntry::FeatureVariation kZCashFeatureVariations[] = {
           kOsDesktop,                                                          \
           FEATURE_VALUE_TYPE(ai_chat::features::kContextMenuRewriteInPlace),   \
       },                                                                       \
-      {                                                                        \
-          "brave-ai-chat-page-content-refine",                                 \
-          "Brave AI Chat Page Content Refine",                                 \
-          "Enable local text embedding for long page content in order to "     \
-          "find "                                                              \
-          "most relevant parts to the prompt within context limit.",           \
-          kOsDesktop | kOsAndroid,                                             \
```

Comment:
flag in iOS needs to be removed too
`ios/browser/flags/about_flags.mm`

Reply:
https://github.com/brave/brave-core/pull/28874/commits/d959d71931660ab14678e05da952c791575c9f4a